src/scan/postgres_seq_scan.cpp

```diff
@@ -14,7 +14,7 @@ PostgresSeqScanGlobalState::PostgresSeqScanGlobalState(Relation relation, duckdb
       m_heap_reader_global_state(duckdb::make_shared_ptr<HeapReaderGlobalState>(relation)), m_relation(relation) {
 	m_global_state->InitGlobalState(input);
 	m_global_state->m_tuple_desc = RelationGetDescr(m_relation);
-	elog(DEBUG3, "-- (DuckDB/PostgresReplacementScanGlobalState) Running %lu threads -- ", MaxThreads());
+	elog(DEBUG3, "-- (DuckDB/PostgresReplacementScanGlobalState) Running %llu threads -- ", MaxThreads());
```

Comment:
This change is actually causing a compiler warning for me. (I'm on Linux)

```
src/scan/postgres_seq_scan.cpp: In constructor ‘pgduckdb::PostgresSeqScanGlobalState::PostgresSeqScanGlobalState(Relation, duckdb::TableFunctionInitInput&)’:
src/scan/postgres_seq_scan.cpp:17:22: warning: format ‘%llu’ expects argument of type ‘long long unsigned int’, but argument 2 has type ‘idx_t’ {aka ‘long unsigned int’} [-Wformat=]
   17 |         elog(DEBUG3, "-- (DuckDB/PostgresReplacementScanGlobalState) Running %llu threads -- ", MaxThreads());
      |                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  ~~~~~~~~~~~~
      |                                                                                                           |
      |                                                                                                           idx_t {aka long unsigned int}
```

Reply:
I think this should work:
```suggestion
	elog(DEBUG3, "-- (DuckDB/PostgresReplacementScanGlobalState) Running " UINT64_FORMAT " threads -- ", MaxThreads());
```

Reply:
Ah that infamous define, yes I've seen it before, I think you're right